src/Symfony/Component/Security/Http/Authenticator/JsonLoginAuthenticator.php

```diff
@@ -45,6 +44,8 @@
  */
 class JsonLoginAuthenticator implements InteractiveAuthenticatorInterface
 {
+    private const MAX_USERNAME_LENGTH = 4096;
+
```

Comment:
I think we should move this to a more generic location, given it's fixing [a CVE](https://symfony.com/blog/cve-2016-4423-large-username-storage-in-session) and used in more authenticators. Maybe as a public constant in a class related to user loading?

Reply:
I moved it to `AuthenticatorInterface` because that's where it is actually used (at input validation time). Makes sense?

Reply:
I've created a PR to move this constant again: https://github.com/symfony/symfony/pull/46584 Let's merge this as-is, I'll rebase mine.